index.html

```diff
@@ -74,23 +124,27 @@ <h2 class="ds-u-font-size--h3 ds-u-font-weight--bold faq-question">
             1. Are Claim and Claim Line Feed (CCLF) files going away?
           </h2>
           <div class="ds-u-font-size--base ds-u-padding-top--1 ds-u-color--white">
-            No, CCLF files are not going away. BCDA is meant to supplement the data that ACOs already receive for the Shared Savings Program or other eligible model participation, which may include CCLF files. Please contact <a href="mailto:SharedSavingsProgram@cms.hhs.gov">SharedSavingsProgram@cms.hhs.gov</a> or your respective model with questions about CCLF files.
+            No, CCLF files are not going away. BCDA supplements the data that participants already receive for the Shared Savings Program (which may include CCLF files). Contact<a href="mailto:SharedSavingsProgram@cms.hhs.gov">SharedSavingsProgram@cms.hhs.gov</a> or your respective model with questions about CCLF files.
```

Comment:
Missing a space here!

```suggestion
            No, CCLF files are not going away. BCDA supplements the data that participants already receive for the Shared Savings Program (which may include CCLF files). Contact <a href="mailto:SharedSavingsProgram@cms.hhs.gov">SharedSavingsProgram@cms.hhs.gov</a> or your respective model with questions about CCLF files.
```